styles/components/alerts.css

```diff
@@ -0,0 +1,34 @@
+.alert-box{
+    width:30rem;
+    padding:0.5rem;
+    border-radius:3px;
+}
+
+.alert-primary{
+    background-color: var(--clr-blue);
+}
+
+.alert-secondary{
+    background-color: var(--clr-grey);
+}
+
+.alert-success{
+    background-color: var(--clr-green);
+}
+
+.alert-danger{
+    background-color: var(--clr-red);
+}
+
+.alert-warning{
+    background-color: var(--clr-orange);
+}
+
+.alert-cancel-btn{
+    float:right;

```

Comment:
Float is old school. You can use other alternatives. Personal opinion though :)

Reply:
Changed the implementation to use flex instead of float 👍🏽